File: src/Shared/Logistics.Shared.Models/InvoiceDto.cs
namespace Logistics.Shared.Models;

public class InvoiceDto
{
    public string Id { get; set; } = default!;
    public ulong LoadRef { get; set; }
    public string LoadId { get; set; } = default!;
    public DateTime CreatedDate { get; set; }
    public CustomerDto Customer { get; set; } = default!;
    public PaymentDto Payment { get; set; } = default!;
}

File: src/Client/Logistics.OfficeApp/src/app/core/api/models/invoice/invoice.dto.ts
import {CustomerDto} from "../customer/customer.dto";
import {PaymentDto} from "../payment/payment.dto";

export interface InvoiceDto {
  id: string;
  loadId: string;
  loadRef: number;
  createdDate: string;
  customer: CustomerDto;
  payment: PaymentDto;
}
